portfolio/src/main/webapp/script.js

```diff
@@ -29,21 +28,47 @@ fetch('footer.html')
     document.querySelector('footer').innerHTML = data;
   });
 
+/**
+ * Fetches stats from the servers and adds them to the DOM.
+ */
+function getData() {
+  fetch('/data').then(response => response.json()).then((names) => {
+    // reference its fields to create HTML content
+    console.log(names);
+    const dataListElement = document.getElementById('data-container');//
+    dataListElement.innerHTML = '';
+    dataListElement.appendChild(
```

Comment:
there is no guarantee you will get back three items... this should be a loop

Reply:
Done!